contrib/exemplar_util/src/main/java/io/opencensus/contrib/exemplar/util/AttachmentValueSpanContext.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.exemplar.util;
+
+import com.google.auto.value.AutoValue;
+import io.opencensus.common.Function;
+import io.opencensus.stats.AttachmentValue;
+import io.opencensus.trace.SpanContext;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * {@link SpanContext} {@link AttachmentValue}.
+ *
+ * @since 0.20
+ */
+@Immutable
+@AutoValue
+public abstract class AttachmentValueSpanContext extends AttachmentValue {
```

Comment:
The idea is we cannot have Trace APIs dependency in Stats APIs, but we can extend the generic abstract class in a contrib util package and have the util depend on both Trace and Stats APIs.

Reply:
That is correct. Maybe put this in comment. Also say we try to avoid having the Trace API ....